tests/Functional/Command/RunCommandTest.php

```diff
@@ -124,7 +128,11 @@ public function testExecuteWithFail()
         $this->assertEquals($intervalTask->getWorkload(), $task->getWorkload());
         $this->assertLessThanOrEqual($intervalTask->getFirstExecution(), $task->getFirstExecution());
         $this->assertLessThanOrEqual($intervalTask->getLastExecution(), $task->getLastExecution());
-        $this->assertEquals($intervalTask->getInterval(), $task->getInterval());
```

Comment:
asserting equals between the interval object resulted in an error: https://github.com/php-task/TaskBundle/runs/7495173211?check_suite_focus=true 
honestly I've no idea why this happens in the CI, locally everything works as expected 😅

Reply:
@Prokyonn is there a __toString method? i would compare at least this

Reply:
OK then all right - should we tag the library first?

Reply:
yeah makes sense to add a version instead of the dev branch before merging :)